### PR TITLE
Add bug report and feature request template

### PR DESCRIPTION
**Pull request by karan6181**
## Description
- Add bug report and feature request template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+labels: bug
+assignees: ''
+
+---
+
+** Environment **
+<!-- Provide the below as necessary -->
+
+- OS: [Ubuntu 20.04]
+- Hardware (GPU, or instance type): [V100]
```
Comment (knighton) [inline .github/ISSUE_TEMPLATE/---bug-report.md:14]: H100

File: .github/ISSUE_TEMPLATE/---feature-request.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+name: "\U0001F680 Feature request"
```
Comment (knighton) [inline .github/ISSUE_TEMPLATE/---feature-request.md:2]: 🚀

**Review by knighton — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/---bug-report.md (hunk 1 above)
Comment (karan6181) [reply .github/ISSUE_TEMPLATE/---bug-report.md]: Done in https://github.com/mosaicml/streaming/pull/48/commits/5ce40e3815fe964e4de8e44ad5f05dc0dc40dc98.

File: .github/ISSUE_TEMPLATE/---bug-report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+name: "\U0001F41B Bug report"
```
[the remark this answers is not in the document]
Comment (karan6181) [reply .github/ISSUE_TEMPLATE/---bug-report.md:2]: Yes, the icon is the same.

File: .github/ISSUE_TEMPLATE/---feature-request.md (hunk 2 above)
Comment (karan6181) [reply .github/ISSUE_TEMPLATE/---feature-request.md:2]: Yes, the icon is the same.

**Pull request merged**